wallet-new/src/Cardano/Wallet/API/V1/Handlers/Accounts.hs

```diff
@@ -92,3 +94,21 @@ redeemAda :: PassiveWalletLayer IO
           -> Handler (WalletResponse Transaction)
 redeemAda _layer _wId _accIdx _redemption =
     error "unimplemented, see [CBR-349]"
+
+getAccountAddresses
+    :: PassiveWalletLayer IO
+    -> WalletId
+    -> AccountIndex
+    -> RequestParams
+    -> FilterOperations WalletAddress
+    -> Handler (WalletResponse AccountAddresses)
+getAccountAddresses _layer _wId _accIdx _pagination _filters =
+    error "unimplemented"
+
+getAccountBalance
+    :: PassiveWalletLayer IO
+    -> WalletId
+    -> AccountIndex
+    -> Handler (WalletResponse AccountBalance)
+getAccountBalance _layer _wId _accIdx =
+    error "unimplemented"
```

Comment:
Ditto as the above. Also, these two endpoints are more integration work for the new-wallet team (cc @edsko ).

Perhaps you folks could help with the integration work?